spec/acceptance/suites/dnfmodule/redis_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'redis', if: %w[centos redhat].include?(os[:family]) && os[:release].to_i >= 8 do
+  redis_name = 'redis'
+
+  it 'runs successfully' do
```

Comment:
Looks like CentOS 9 doesn't have a Redis 6 module (Redis 6.2.7 is a regular package).